common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4012,6 +4013,15 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(32)
 
+  val WORKER_READBUFFERS_CHECK_THREAD_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.worker.readBufferDispatcherThreadWatchdog.checkInterval")
+      .categories("worker")
+      .version("0.5.2")
+      .internal
```

Comment:
Why is this config internal?

Reply:
This check is not necessary after we told the users to enable it. Because after this PR, this thread should not exit unexpectedly.